scripts/prepublish.js

```diff
@@ -83,7 +83,8 @@ function getPipeline () {
 function getWorkflowId (pipeline) {
   return fetch(`pipeline/${pipeline.id}`)
     .then(response => {
-      const workflow = response.data.workflows[0]
+      const workflows = response.data.workflows
```

Comment:
are these always in a deterministic order? do they need any kind of sorting?

Reply:
Good point. Will sort to be safe.

Reply:
Done. Also added a check to wait for running workflows to finish.